guardrails/validator_base.py

```diff
@@ -329,6 +329,10 @@ def validate_stream(
         # if validate doesn't set validated chunk, we set it
         if validation_result.validated_chunk is None:
             validation_result.validated_chunk = chunk_to_validate
+        if isinstance(validation_result, FailResult):
+            if validation_result.error_spans is None:
```

Comment:
collapse - 

`if isinstance(validation_result, FailResult) and validation_result.error_spans is None`